golem-component-service-base/tests/db/migration/postgres/001__init.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE components
+(
+    namespace           text    NOT NULL,
+    component_id        uuid    NOT NULL PRIMARY KEY,
+    name                text    NOT NULL,
+    UNIQUE (namespace, name)
```

Comment:
This index can't be used to search by name without namespace. If we have such operation - please add non-unique index by name